assets/component-menu-drawer.css

```diff
@@ -263,3 +263,11 @@ details[open].menu-opening > .menu-drawer__submenu {
 .menu-drawer .list-social__link {
   padding: 1.1rem 1.1rem;
 }
+
+@media screen and (max-width: 749px) {
+  .menu-drawer.country-selector-open {
+    transform: none !important;
+    filter: none !important;
+    transition: none !important;
+  }
+}
```

Comment:
I don't think this was meant to be applied for the menu drawer country/language selectors but for the footer one 🤔

Reply:
One of the UX feedbacks we received regarding the country selector in the menu drawer is that we should also overlay the header so that we don't see 2 close buttons, since clicking the menu drawer's close button would also close the country selector and this might be a source of confusion. These styles allow us to do that. 

<img width="286" alt="Screenshot 2023-12-14 at 8 45 30 AM" src="https://github.com/Shopify/dawn/assets/54807699/627d68c9-9c6a-400d-8703-41c954c23333">